docs/Windows.md

```diff
@@ -1,43 +1,172 @@
 ### Getting Started (Windows)
 
-#### Downloading the nightlies
+#### The Windows SDK and the Native Tools Command Prompt
 
-1. We assume you are reading these instructions on the Azure DevOps page for windows-swift at <https://dev.azure.com/compnerd/windows-swift>.
-2. Choose `Pipelines` > `builds` from the left of the dashboard.
-3. Click the most recent successful build.
-4. At the top right of this page to the right of where it says `All logs` there is a three dot button. Under this choose `Artifacts` > `toolchain` to download the nightly toolchain to your machine.
+You will need an installation of the Windows SDK to develop with the Swift toolchain described here. An easy way to get the Windows SDK is to install Visual Studio, currently Visual Studio 2017 is needed. The following instructions suppose that you have Visual Studio 2017 installed. This will also make the  `x64 Native Tools Command Prompt for VS 2017` available, it should be accessible from the `Visual Studio 2017` folder in the Start menu.
```

Comment:
The "is to install Visual Studio", "currently Visual Studio 2017 is needed" is a comma splice, perhaps use a semicolon?  It really should be Visual Studio 2017 or newer though, as Visual Studio 2019 works just fine (arguably works better).

There is an extra space after the "make the" and `x64 Native Tools Command Prompt for VS 2017`.